src/node_api.cc

```diff
@@ -2054,6 +2053,49 @@ napi_status napi_instanceof(napi_env e,
     return napi_set_last_error(napi_function_expected);
   }
 
+  napi_value value, key, jsRes;
+  napi_status status;
+  napi_valuetype valueType;
+
+  // Get "Symbol" from the global object
+  status = napi_get_global(e, &value);
+  if (status != napi_ok) return status;
```

Comment:
I used the pattern from here: https://github.com/nodejs/abi-stable-node/blob/api-prototype-8.x/src/node_api.cc#L753